src/odemis/acq/fastem.py

```diff
@@ -531,15 +321,17 @@ def run(self):
                           int((1 - self._roa.overlap) * self._multibeam.resolution.value[1]))
         self._detector.updateMetadata({model.MD_FIELD_SIZE: eff_field_size})
 
+        self._detector.updateMetadata({model.MD_SLICE_IDX: self._roa.slice_index.value})
+        self._detector.updateMetadata({model.MD_USER: self._username})
```

Comment:
Right, the technolution driver does nothing with the the MD_USER metadata. I am just updating it so that if a bugreport is its value is correct in `odemis-hw-status`.